utils.py

```diff
@@ -0,0 +1,7 @@
+from datetime import datetime
+
+
+# Add to log
+def log(logger, text):
+    logger.info("{} {}".format(datetime.now(), text))
```

Comment:
Acho que não precisamos dessa função. O logger já adiciona data e hora e mais várias outras informações na mensagem logada, e dá pra configurar o formato também.

Reply:
@rougeth Sério? Então to usando errado a vida toda..